ebuild-maintenance/package-moves/text.xml

```diff
@@ -113,6 +115,48 @@ you have updated every file in <c>profiles/</c> directory that
 happens to contain an entry which may be affected by your change.
 </p>
 
+</body>
+</section>
+
+<section>
+<title>Updating prior update entries when moving the package again</title>
+<body>
+
+<p>
+When the same package is moved again, the previous update entries should
+be updated appropriately. This is meant to make the situation more transparent
+to users reading update entries and to ensure that the process is handled
+efficiently even if the package manager does not implement updates in a robust
+way. This involves the following steps:
+</p>
+
+<ol>
+  <li>
+    The previous package moves for the package in question must be updated
+    to reference the final name. That is, rather than the chain A → B → C,
+    we want to have two update entries: A → C, and B → C.
+  </li>
+
+  <li>
+    If the package is being moved to a name that it used before, the original
+    move entry must be removed. That is, rather than the chain A → B → A,
+    we want to have the reverse entry: B → A. If the package manager did not
+    move A → B before, we don't want it to touch the package at all.
+  </li>
+
+  <li>
+    As a combination of the two aforementioned steps, a chain of A → B → C → A
+    would be replaced by two moves: B → A, and C → A.
+  </li>
+
+  <li>
+    If the package was slot-moved before, the slot moves should be updated
+    to use the final package name, and moved after the final package move.
+    That is, rather than the chain: A:S₁ → A:S₂, A → B; we prefer to have
+    the chain: A → B, B:S₁ → B:S₂.
```

Comment:
The order in which moves are executed isn't specified by PMS. For example, there's no defined order for A → B in `4Q-2023` and B:S₁ → B:S₂ in `1Q-2024`.

Reply:
Not even if they're in the same file? Should we specify slotmoves for both old and new package name then?

Reply:
Within the same file should be fine, so maybe explicitly say so?
https://archives.gentoo.org/gentoo-pms/message/4fde4acef1dd0c85e82f23dfb5ae0322

Reply:
Updated to say that all entries must be in the same file.